src/main/java/com/vackosar/gitflowincrementalbuild/boundary/Configuration.java

```diff
@@ -70,6 +77,23 @@ public Configuration(MavenSession session) throws IOException {
         }
     }
 
+    private Map<String, String> parseSpaceDelimitedArgs(String value) {
+        return value.isEmpty()
+                ? Collections.emptyMap()
+                : Arrays.stream(value.split(" "))
+                    .map(String::trim)
+                    .filter(s -> !s.isEmpty())
+                    .map(this::keyValuePairToArray)
+                    .collect(Collectors.toMap(a -> a[0], a -> a[1], (a, b) -> a, LinkedHashMap::new));
+    }
+
+    private String[] keyValuePairToArray(String pair) {
```

Comment:
there is probably some method for parsing jvm arguments somewhere in the standard library

Reply:
None that I know of.